wbooks/src/app/components/Book/styles.ts

```diff
@@ -0,0 +1,38 @@
+import { StyleSheet } from 'react-native';
+import { titleColor, authorColor, white } from '@constants/colors';
+
+const styles = StyleSheet.create({
+  title: {
+    left: '115%',
+    lineHeight: 20,
+    textAlign: 'center',
+    color: titleColor,
+    fontSize: 13
+  },
+  author: {
+    right: '350%',
+    top: 20,
+    lineHeight: 18,
+    textAlign: 'center',
+    color: authorColor,
+    fontSize: 12
+  },
+
+  bookContainer: {
+    display: 'flex',
+    backgroundColor: white,
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    marginTop: 45,
+    marginLeft: 18
+  },
+  textContainer: {
+    display: 'flex',
+    flexDirection: 'row',
```

Comment:
why are you setting this as row? the text container should be column, since it goes in vertical direction